File: exercicio-IC/exemplo.cs
using System;

namespace MediaNotas {
    class Program {
        static void Main(string[] args) {
            int numAlunos;
            float nota, soma = 0;

            Console.Write("Digite o número de alunos: ");
            numAlunos = int.Parse(Console.ReadLine());

            // Loop para ler as notas dos alunos
            for (int i = 1; i <= numAlunos; i++) {
                Console.Write($"Digite a nota do aluno {i}: ");
                nota = float.Parse(Console.ReadLine());
                soma += nota; // Acumula as notas para calcular a média
            }

            float media = soma / numAlunos; // Calcula a média
            Console.WriteLine($"A média das notas é: {media:F2}");

            if (media < 5) {
                Console.WriteLine("Atenção: Alunos reprovados!");
            }
        }
    }
}

File: exercicio-IC/exemplo.py
num_alunos = int(input("Digite o número de alunos: "))
soma = 0

# Loop para ler as notas dos alunos
for i in range(1, num_alunos + 1):
    nota = float(input(f"Digite a nota do aluno {i}: "))
    soma += nota  # Acumula as notas para calcular a média

media = soma / num_alunos  # Calcula a média
print(f"A média das notas é: {media:.2f}")

if media < 5:
    print("Atenção: Alunos reprovados!")
